src/config/entities/configuration.ts

```diff
@@ -135,20 +135,21 @@ export default () => ({
     apiKey: process.env.EMAIL_API_KEY,
     fromEmail: process.env.EMAIL_API_FROM_EMAIL,
     fromName: process.env.EMAIL_API_FROM_NAME || 'Safe',
-    templates: {
-      recoveryTx: process.env.EMAIL_TEMPLATE_RECOVERY_TX,
-      unknownRecoveryTx: process.env.EMAIL_TEMPLATE_UNKNOWN_RECOVERY_TX,
-      verificationCode: process.env.EMAIL_TEMPLATE_VERIFICATION_CODE,
-    },
-    verificationCode: {
-      resendLockWindowMs: parseInt(
-        process.env.EMAIL_VERIFICATION_CODE_RESEND_LOCK_WINDOW_MS ??
-          `${30 * 1000}`,
-      ),
-      ttlMs: parseInt(
-        process.env.EMAIL_VERIFICATION_CODE_TTL_MS ?? `${5 * 60 * 1000}`,
-      ),
-    },
+    // TODO: Decide whether we reuse these when implementing the email service
```

Comment:
Nitpick: not an strong opinion, but I'd more in favour of deleting this code instead of commenting it. We can back to the commits history if we need to reuse this. Wdyt?

Reply:
Removed in 7e7818d428d0dd2c3ca865b73be068cdb2d517e5.